openapi-schema.json

```diff
@@ -10280,17 +10608,19 @@
             "required": true,
             "schema": {
               "type": "string",
-              "description": "Component Id"
+              "description": "Component Id",
+              "example": "801B800245E468A52AEBEC4BE31CFF5AF82F371DAEF5F158AC2E98C2FA324B46"
             }
           },
           {
             "name": "account",
             "in": "query",
-            "description": "Account",
+            "description": "Account ( Should be in Format - AccountName.ACCT )",
             "required": true,
             "schema": {
               "type": "string",
-              "description": "Account"
+              "description": "Account ( Should be in Format - AccountName.ACCT )",
+              "example": "Account Name"
```

Comment:
minor: shouldn't the example follow the format mentioned in description?

Reply:
Discussed in offline. Working on this enhancement in the current sprint.